charts/secrets-store-csi-driver/README.md

```diff
@@ -0,0 +1,47 @@
+# secrets-store-csi-driver
+
+## Installation with Helm 3
+
+Quick start instructions for the setup and configuration for secrets-store-csi-driver using Helm.
+
+### Prerequisites
+
+- [Helm v3.0+](https://helm.sh/docs/intro/quickstart/#install-helm)
+
+### Install charts
+
+**Get the source**
+```bash
+$ go get -d sigs.k8s.io/secrets-store-csi-driver
+$ cd "$(go env GOPATH)/src/sigs.k8s.io/cluster-api-provider-azure"
```

Comment:
wrong project 😄

Reply:
My bad! Was copying the last command from my terminal 😆

Reply:
thank you, updated it!